### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity###

*Description of changes:*
Report fix
Temporary fix to create the document meta instance in the filings endpoint. I will be refactoring the document_meta to be a helper/utility class and remove the  instance variables in a subsequent PR

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/src/legal_api/reports/report.py [hunk 1]
```diff
@@ -176,10 +176,11 @@ def _set_dates(self, filing):
         filing['effective_date_time'] = effective_date.strftime(f'%B %-d, %Y at {effective_hour}:%M %p Pacific Time')
         filing['effective_date'] = effective_date.strftime('%B %d, %Y')
         # Recognition Date
-        recognition_datetime = LegislationDatetime.as_legislation_timezone(self._business.founding_date)
-        recognition_hour = recognition_datetime.strftime('%I').lstrip('0')
-        filing['recognition_date_time'] = \
-            recognition_datetime.strftime(f'%B %-d, %Y at {recognition_hour}:%M %p Pacific Time')
+        if self._business:
```
Comment (severinbeauvais) [inline legal-api/src/legal_api/reports/report.py:179]: This is just to support testing right?

Comment (lekshmimallika-aot) [reply legal-api/src/legal_api/reports/report.py:179]: No, this is the fix for the issue you mentioned yesterday.. unable to download IA report for a paid filing

Comment (severinbeauvais) [reply legal-api/src/legal_api/reports/report.py:179]: How is it that the other code in this file that de-references self._business doesn't bomb?
Eg, lines 137, 144 and 275

Comment (lekshmimallika-aot) [reply legal-api/src/legal_api/reports/report.py:179]: Please see the condition above the specified line numbers

Comment (severinbeauvais) [reply legal-api/src/legal_api/reports/report.py:179]: OK. Resolving this conversation.

**Pull request merged**